npm install is failed, because react-router want history@1.13.x

```
npm ERR! node v4.2.2
npm ERR! npm  v2.14.7
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package history@1.16.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-router@1.0.2 wants history@1.13.x
```

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "ejs": "^2.3.4",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.9.1",
-    "history": "^1.9.0",
+    "history": "1.13.x",
     "humps": "^0.6.0",
     "isomorphic-fetch": "^2.1.1",
     "jsdom": "^7.0.1",

